src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -22,6 +22,7 @@ const AssigneeSelector = React.createClass({
 
   mixins: [
     Reflux.listenTo(GroupStore, 'onGroupChange'),
+    Reflux.connect(MemberListStore, 'memberList'),
```

Comment:
Does`MemberListStore` need a `getInitialState` method?

https://github.com/reflux/refluxjs/tree/v0.4.1/#using-refluxconnect

Reply:
do you think this is safe to merge in that case?

Reply:
Should we set initial state of `memberList` in the component to an empty array?

Reply:
i could be wrong, but I believe this covers it and works better in the case of the store already being populated when the component mounts: https://github.com/getsentry/sentry/blob/assignee-race-condition/src/sentry/static/sentry/app/stores/memberListStore.jsx#L7